examples/6_custom_ssh_config/main.go

```diff
@@ -0,0 +1,125 @@
+// this is the simplest example possible
+package main
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"github.com/nornir-automation/gornir/pkg/gornir"
+	"github.com/nornir-automation/gornir/pkg/plugins/connection"
+	"github.com/nornir-automation/gornir/pkg/plugins/inventory"
+	"github.com/nornir-automation/gornir/pkg/plugins/logger"
+	"github.com/nornir-automation/gornir/pkg/plugins/output"
+	"github.com/nornir-automation/gornir/pkg/plugins/runner"
+	"github.com/nornir-automation/gornir/pkg/plugins/task"
+	"golang.org/x/crypto/ssh"
+)
+
+func getPubKeySigner(host *gornir.Host, sshPrivKeyFname string, logger gornir.Logger) (*ssh.Signer, error) {
+	key, err := ioutil.ReadFile(sshPrivKeyFname)
+	if err != nil {
+		logger.Debug(fmt.Sprintf("unable to read private key: %v", err))
+		return nil, err
+	}
+
+	// Create the Signer for this private key.
+	signer, err := ssh.ParsePrivateKey(key)
+	if err != nil {
+		logger.Error(fmt.Sprintf("unable to parse private key: %v", err))
+		return nil, err
+	}
+	return &signer, nil
+}
+
+func GetSSHConfig(host *gornir.Host, logger gornir.Logger) (*ssh.ClientConfig, error) {
+	var authMethods = []ssh.AuthMethod{ssh.Password(host.Password)}
+	// Under normal circumstances, you probably want to use something like the github.com/kevinburke/ssh_config package
```

Comment:
I'd refrain from having examples like this in comments. If we want to add it I suggest we have a full working example but I personally think it'd be too much and removing the comments should be good.